src/resources/Account.js

```diff
@@ -25,7 +25,7 @@ export default class Account extends Resource {
     })
   }
 
-  async login (email, password = '') {
+  async login (email, password = '', backofficeLogin = false) {
```

Comment:
Would prefer these params to be named: `login(email, password, backoffice)` as "Login" is implied.